exec/java-exec/src/test/java/org/apache/drill/exec/dotdrill/TestDotDrillUtil.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.dotdrill;
+
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+import org.apache.drill.exec.store.dfs.DrillFileSystem;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TestDotDrillUtil {
+
+  private static final String TEST_ROOT_DIR = System.getProperty("java.io.tmpdir") + "/dot_drill_util";
```

Comment:
I think you may want to look at some existing tests to inherit baseclass like BaseTestQuery. This will make sure that the test directories are cleaned up properly.

@ilooner  Can you please take a look at the test file.

Reply:
@ilooner Could you please review the Test file. This test scenario is more related to FileSystem related tests. Please let me know if there are any existing base class that can be used here.